src/uwtools/yaml_file.py

```diff
@@ -0,0 +1,81 @@
+import yaml
```

Comment:
The imports should probably come after the doc block.

Reply:
done